### DISCUSSION TITLE
Web or Mobile Development?

### DISCUSSION THREAD

**Discussion by devIgor1** [drknzz/GitHub-Achievements#284] [Q&A] [answered]
Hey guys, do you prefer web or mobile development? And why?

**Comment by viniarski** [comment, marked as answer]
Depends of the region, this might help:
https://gs.statcounter.com/platform-market-share/desktop-mobile-tablet/worldwide

**Comment by ojasvinaik13** [comment]
I prefer web development! This is because websites can be accessed from any device with a web browser, including desktops, laptops, tablets, and smartphones. Developing a website ensures compatibility across multiple platforms without the need to develop separate versions for each platform.
